Flutter/lib/pages/setting.dart

```diff
@@ -11,257 +16,235 @@ class Setting extends StatefulWidget {
-                                value: PropertyNameSortingType.none,
-                                child: Text("保持原样"),
-                              ),
-                              DropdownMenuItem(
-                                value: PropertyNameSortingType.ascending,
-                                child: Text("升序排列"),
-                              ),
-                              DropdownMenuItem(
-                                value: PropertyNameSortingType.descending,
-                                child: Text("降序排序"),
-                              ),
-                            ],
-                            onChanged: (value) {
-                              setState(() {
-                                ConfigHelper().config.propertyNameSortingType =
-                                    value;
-                                controller.orderPropeties();
-                              });
-                            },
-                          ),
-                          Text("属性排序"),
-                        ],
-                      )),
-                  Container(
-                      margin: EdgeInsets.all(10.0),
-                      padding: EdgeInsets.all(5.0),
-                      decoration: BoxDecoration(
-                          border: Border.all(color: Colors.black, width: 1.0)),
-                      child: Row(
-                        mainAxisSize: MainAxisSize.min,
-                        children: <Widget>[
-                          Checkbox(
-                            value: ConfigHelper().config.addMethod,
-                            onChanged: (value) {
-                              setState(() {
-                                ConfigHelper().config.addMethod = value;
-                              });
-                            },
-                          ),
-                          Text("添加保护方法"),
-                        ],
-                      )),
-                ],
+                );
+              },
+            );
+          },
+        ),
+        TapButton(
+          title: "保存配置",
+          icon: Icons.save,
+          onPressed: () {
+            ConfigHelper().save();
+            showToast("保存配置成功");
+          },
+        ),
+        TapButton(
+          title: "生成Dart",
+          icon: Icons.flag,
+          onPressed: () {
+            controller.generateDart();
+          },
+        ),
+      ],
+    );
+    return Container(
+      padding: EdgeInsets.symmetric(vertical: 6),
+      child: settingRow,
+    );
+  }
+}
+
+class MoreSetting extends StatefulWidget {
+  final JsonToDartController controller;
+
+  const MoreSetting({Key key, this.controller}) : super(key: key);
+  @override
+  _MoreSettingState createState() => _MoreSettingState();
+}
+
+class _MoreSettingState extends State<MoreSetting> {
+  JsonToDartController get controller => widget.controller;
+  @override
+  Widget build(BuildContext context) {
+    var buttonGroup = Wrap(
```

Comment:
These lines can be moved out of the `build` methods since it has hundreds of rows.